### ISSUE TITLE
Quick backup does not remember settings

### ISSUE THREAD

**Issue by blakeplusplus** [psenough/NPlusPlusAssistant#3]
Quick backup would greatly benefit from being able to remember your settings. Or at least, changing the defaults from just profile and editor levels to add attract files would be great.

**Comment by psenough** [comment]
yeah that sounds useful and relatively easy to implement, i'll add it to the possible new feature todo list
